packages/adp-tooling/test/unit/client/target-systems.test.ts

```diff
@@ -46,135 +80,114 @@ describe('getEndpointNames', () => {
 
 describe('TargetSystems', () => {
     let targetSystems: TargetSystems;
-    const endpoints: Endpoint[] = [
-        { Name: 'SystemA', Client: '010', Url: 'http://systema.com', Authentication: 'Basic' },
-        { Name: 'SystemB', Client: '200', Url: 'http://systemb.com', Authentication: 'NoAuthentication' }
-    ];
+    // const endpoints: Endpoint[] = [
+    //     { Name: 'SystemA', Client: '010', Url: 'http://systema.com', Authentication: 'Basic' },
+    //     { Name: 'SystemB', Client: '200', Url: 'http://systemb.com', Authentication: 'NoAuthentication' }
+    // ];
```

Comment:
remove?

Reply:
Removed in: [6cf7508](https://github.com/SAP/open-ux-tools/pull/3031/commits/6cf7508a4afbaae9013471464fe53671acf64869)